### PR TITLE
Adding CoAP options to js server

### PR DESCRIPTION
**Pull request by hquynguyen**
Hello,
I'm Quy, working on S5 project "SCHC on LwM2M". I'd like to update code of adding CoAP options in schc.js as well as fixing  some bugs I've found.
Thanks

### CODE REVIEW

**Comment by tkerdonc** [comment]
Thanks for the contribution, and sorry about the delay, must have missed a notification.
This can be merged, but could you rephrase your commit message, maybe just putting "Adding CoAP options to js server" and detailing briefly detailing the bug fixes you added ?

**Comment by hquynguyen** [comment]
Hello, sorry for the delay because I had been busy for the exams.
I edited the title.
About the issues:
- In case of handling string value in valueSent action
```js
for (var l = 0; l < length; l++) {
		for (var b = 7; b >= 0; b--) {
		    bit = FV & (1 << b)
		    that.seteBufBit(bin)
		}
}
```
The idea is getting each character in FV then convert to bit. So in the for loop, it should be FV.length, and bit = FV.charCodeAt(i) & (1<<b)
- Another small issue in python/SCHC/Decompressor.py
```python
buff.add_bit( headers.next_bit() )
```
should be buf.add_bit

**Comment by tkerdonc** [comment]
Thanks for the details, merging the changes.
